app/transformers/api/current/medical_history_transformer.rb

```diff
@@ -8,7 +8,12 @@ def to_response(medical_history)
     end
 
     def from_request(medical_history_payload)
-      Api::Current::Transformer.from_request(medical_history_payload)
+      Api::Current::Transformer.from_request(medical_history_payload).tap do |params|
+        if params[:hypertension].blank?
```

Comment:
This is fine, but keep in mind that `v2` ([which inherits from Current](https://github.com/simpledotorg/simple-server/blob/master/app/transformers/api/v2/medical_history_transformer.rb)) will also behave this way. Since this is an accretion `v3` already has versions of itself that don't handle `hypertension`, so this check is valuable and good.

However, we should check with the mobile team, in the `to_response` [case](https://github.com/simpledotorg/simple-server/blob/master/app/transformers/api/v2/appointment_transformer.rb#L4), whether we need to do an `except` for the `hypertension` field for `v2` or will the old API continue to work correctly (as a sanity check).

Reply:
Ooh I did not know that V2 inherits from Current. This is good to know. I'll check with the mobile team and update the necessary transformer accordingly. 👍